services/resource-service/server/src/main/java/org/apache/airavata/mft/resource/server/backend/file/FileBasedResourceBackend.java

```diff
@@ -90,10 +90,10 @@ public boolean deleteSCPStorage(SCPStorageDeleteRequest request) {
                         JSONObject r = (JSONObject) resource;
 
                         SCPStorage storage = SCPStorage.newBuilder()
-                                .setStorageId(((JSONObject)r.get("scpStorage")).get("storageId").toString())
-                                .setHost(((JSONObject)r.get("scpStorage")).get("host").toString())
-                                .setUser(((JSONObject)r.get("scpStorage")).get("user").toString())
-                                .setPort(Integer.parseInt(((JSONObject)r.get("scpStorage")).get("port").toString())).build();
+                                .setStorageId(((JSONObject) r.get("scpStorage")).get("storageId").toString())
```

Comment:
Is this an intentional change? Seems like not relevant for this PR

Reply:
Hello,

So this is one of the indentation changes produced by our IDE. Do you want us to revert it?

Reply:
Yes please do. Try to keep the changes as much as relevant to the PR